UnityProject/Assets/Scripts/Gateway/StationGateway.cs

```diff
@@ -78,11 +77,19 @@ protected virtual void UpdateMe()
 				DetectPlayer();
 				timeElapsedServer = 0;
 			}
+
+			timeElapsedServerSound += Time.deltaTime;
+			if (timeElapsedServerSound > 7 && isOn)
```

Comment:
Why 7? Try to get length from the sound if that is possible or at least make a constant variable for it.

Reply:
Dont think getting sound length is possible